class Error2(Exception):
    pass